# An array of pose hypothesis 

Header header

geometry_msgs/Pose2D[] leftDot
geometry_msgs/Pose2D[] rightDot

geometry_msgs/Pose2D[] leftDistortDot
geometry_msgs/Pose2D[] rightDistortDot
